src/styles/GameScreen.css

```diff
@@ -0,0 +1,48 @@
+*{
```

Comment:
Generally speaking I'd recommend avoiding using this selector as it can have unintended consequences. If this is intended just to remove the `margin: 8px` added to the `body` element by the browser's default stylesheet then I'd recommend just using 
```css
body {
  margin: 0;
}
```
instead